### ISSUE TITLE
Does coc-explorer support "directory bookmark" like NERDTree does?

### ISSUE THREAD

**Issue by yanlinsun** [weirongxu/coc-explorer#228]
I prefer to bookmark my frequent access directories when I use nerdtree. 
coc-explorer is great and I intend to switch to it completely. However, I search a lot but not found any information about the "directory bookmark" feature.

It is similar to set some directories as favorite. Like:
- [Favorite Directories]
  + Vim (~/vimfiles)
  + home (~)
  + Workspace (~/workspaces)
  + OS Settings (/etc)
  + Project1 (~/workspaces/project1)

And if I hit 'o' on any of the entries, the root folder automatically jumps to it.

**Comment by weirongxu** [comment]
Not supported yet, might need `coc-bookmark` to support this first.

I inform the @voldikss to see if the author intends to support this

**Comment by voldikss** [comment]
/assign @voldikss 
I will implement this when I have free time.

**Comment by yanlinsun** [comment]
@voldikss @weirongxu Thank you for accepting this enhancement. I would like to implement this feature if you don't mind. 

Before that, I have a second opinion on which project this feature belongs to. I think it should go to coc-explorer rather than coc-bookmark.
1. The name "directory bookmark" is only explanatory. It does not imply that it is one of the coc-bookmark features.
2. It is about files and directories, which is different from the definition of a bookmark.
3. Consider similar file manager products, like NERDTree, Finder, File Explorer, they all support this feature; no need an external plugin to enable it.
4. When creating such "bookmark", the user will perform the action in the coc-explorer buffer not a buffer of a concrete file.

**Comment by weirongxu** [comment]
@yanlinsun 
I agree with your idea that the directories bookmarks should be deeply associated with the explorer. And we should have some actions built-in the explorer. Dependent on bookmarks is not the best implementation

**Comment by voldikss** [comment]
I also agree with @yanlinsun 's view. 👍

**Comment by yanlinsun** [comment]
Great. @weirongxu I am happy to help to implement it if you already started. In the meantime,  I will folk and start from there.

**Comment by weirongxu** [comment]
> Great. @weirongxu I am happy to help to implement it if you already started. In the meantime, I will folk and start from there.

I have to do something else. And I haven't time to add any new features recently, because I planned to refactor the floating window.
